.github/workflows/http-dotnet-httpclient.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           name: drop
           path: |
             ${{ env.relativePath }}/src/bin/Release/*.nupkg
+            ${{ env.relativePath }}/Microsoft.Kiota.Http.HttpClient.Tests/TestResults
```

Comment:
can you upload as a separate artifact to avoid mixing concerns please?

Reply:
Agreed. Will action and resolve.

Reply:
Resolved via https://github.com/microsoft/kiota/pull/522/commits/b52a2a79c2715c57a982e41b7682f338a4045d44